cluster/gce/config-default.sh

```diff
@@ -481,5 +481,8 @@ WINDOWS_NODE_TAINTS="${WINDOWS_NODE_TAINTS:-node.kubernetes.io/os=win1809:NoSche
 # Whether to set up a private GCE cluster, i.e. a cluster where nodes have only private IPs.
 GCE_PRIVATE_CLUSTER="${KUBE_GCE_PRIVATE_CLUSTER:-false}"
 
+# Whether to enable Windows nodes. Enabled by default.
+GCE_WINDOWS_NODES="${KUBE_GCE_WINDOWS_NODES:-true}"
```

Comment:
Why do we need it? If you set NUM_WINDOWS_NODES=0 it also doesn't create windows nodes?

Reply:
This PR is from some time ago and IIRC this wasn't the case. But I'll have a look to double check that.

Reply:
It's actually not needed. Replaced the condition with NUM_WINDOWS_NODES.